pkg/gitreceive/k8s_util.go

```diff
@@ -108,9 +108,12 @@ func slugbuilderPod(
 	pod.Spec.Containers[0].Name = slugBuilderName
 	pod.Spec.Containers[0].Image = image
 
+	if _, ok := env["DEIS_DISABLE_CACHE"]; !ok {
```

Comment:
@kmala since cache is enabled by default and benefits most (if not all) projects using the slugbuilder, I chose to enable it by default. I'll dedicate a section in the App configuration section inside the docs on how to disable cache and also about the `CACHE_PATH` variable.